src/pal/inc/pal_corefxnet.h

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+/*++
+
+Module Name:
+
+    pal_corefxnet.h
+
+Abstract:
+
+    Header file for functions meant to be consumed by the CoreFX Net libraries.
+
+--*/
+
+#include "pal.h"
+
+#ifndef __PAL_COREFX_NET_H__
+#define __PAL_COREFX_NET_H__
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+#define COREFX_NET_SUCCESS 0x00000000
+#define COREFX_NET_INVALID_PARAM 0xc000000d
+
+  
+  struct PalIPAddress {
+    UCHAR isIPV4;
+    INT port;
+    UINT scopeId;
+    UCHAR bytes[16];
+  };
+
+  PALIMPORT
+  INT
+  PalGetIPAddresses(const CHAR* hostName, // host name
```

Comment:
The naming convention used for other CoreFX helpers is to have just a simple name, without any pal prefix. We should be consistent.